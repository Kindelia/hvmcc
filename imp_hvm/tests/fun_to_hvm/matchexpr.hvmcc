function test_match (a) {
    {test_match a} = match a {
        {SomeCtr} => Unit
        0 => 1
        a => a
    }
}