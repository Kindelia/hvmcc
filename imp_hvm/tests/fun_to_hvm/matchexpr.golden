(test_match a) = (test_match.0 a)
(test_match.0 (SomeCtr)) = (Unit)
(test_match.0 0) = 1
(test_match.0 a) = a